mantidproject/mantid
# Add BondAnalysis mode in SimulatedDensityOfStates

Fixes #13165.

To test:
- Use the CASTEP and .phonon files in `Babylon5/Public/DanNixon/CASTEP/ba`
- Run `SimulatedDensityOfStates` in BondAnalysis spectrum type
- See that there are a significant number of stretching modes found in the higher mode numbers

Since this feature is still somewhat experimental it is not mentioned in the release notes.
